tests/e2e/helper/helper_oc.go

```diff
@@ -93,5 +93,17 @@ func waitForCmdOut(cmd string, timeout time.Duration, errOnFail bool, check func
 			}
 		}
 	}
+}
 
+// AfterFailed runs certain oc commands like "oc get pod",
+// "oc get dc" and "oc get dc -o yaml" to analyze failure cause
+func AfterFailed() {
```

Comment:
There is no connection as the function is isolated among different packages implementation of [e2e/helper](https://github.com/openshift/odo/pull/1632/files#diff-6b15e7b1bbc67ef4b8f395c12cdd7206R100) and [e2escenarios/helper](https://github.com/openshift/odo/pull/1632/files#diff-7b297e240bff316300c5bd8cc03238ffR112) due to the current structure. Pr #1612 will fix such issue because testsuite is going to use a common helper package.